src/ray/common/scheduling/cluster_resource_scheduler.h

```diff
@@ -149,6 +159,9 @@ class ClusterResourceScheduler {
       const std::string &local_node_id,
       const std::unordered_map<std::string, double> &local_node_resources);
 
+  // Mapping from predefined resource indexes to resource strings
+  std::string GetResourceNameFromIndex(int64_t res_idx);
```

Comment:
This doesn't seem like it needs to be a class method. Please make it a helper function in the .cc file and do not expose it in the header file.

Reply:
Not, it needs as I am using it in the node_manager.cc